2017/Day 2-1.py

```diff
@@ -0,0 +1,10 @@
+
+checksum = 0
+with open("Day 2 - input", "r") as file:
+    for line in file:
+        row = [int(i) for i in line.split('\t')]
```

Comment:
The separator in `line.split()` can be removed as `split()` splits on all white space by default